frontend/src/components/projectDetail/liveViewButton.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { FormattedMessage } from 'react-intl';
+
+import messages from './messages';
+import { CustomButton } from '../button';
+
+export const LiveViewButton = ({ projectId, className, compact = false }: Object) => (
```

Comment:
Hi Emi, I see you using Object Type in this section and wanted to know if it's necessary as this file is not typescript and Object type is too vague for the types, isn't it?

Reply:
You're right, it can be removed, I think I just used code from another view as a template and it was there :)